packages/react-components/src/components/Link/ButtonLink.jsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { StyledATag } from '.'
+
+const StyledNavLink = styled(StyledATag).attrs(props => ({
+  px: 3,
+  py: 2,
+  mx: 1,
+  fontSize: 2,
+  cursor: props.disabled ? 'not-allowed' : 'pointer',
+  color: props.theme.colors.buttons.secondary.borderColor
+}))`
+  /* We couldn't set the border above. So add it here instead. */
```

Comment:
When trying to set the border above, it gets clobbered by the default link styles. I can remove this comment to keep this clean. But wanted to check if this is fine to do it this way.